Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -20,6 +20,8 @@ extension DotLottieFile {
       dotLottieCache: DotLottieCacheProvider? = DotLottieCache.sharedCache)
       -> Result<DotLottieFile, Error>
     {
+      assert(!Thread.isMainThread, "Calling this API in the main thread will affect the performance of your App")
```

Comment:
This should use `LottieLogger.shared.assert`

```suggestion
      LottieLogger.shared.assert(!Thread.isMainThread, "`DotLottieFile.SynchronouslyBlockingCurrentThread` methods shouldn't be called on the main thread.")
```